plugins/bazaar/src/components/EntityBazaarInfoCard/EntityBazaarInfoCard.tsx

```diff
@@ -0,0 +1,345 @@
+
+const useStyles = makeStyles({
+  description: {
+    wordBreak: 'break-word',
+  },
+  icon: {
+    marginRight: '1.75rem',
+  },
+  link: {
+    color: '#9cc9ff',
+    '&:hover': {
+      textDecoration: 'underline',
+    },
+  },
+  memberLink: {
+    display: 'block',
+    marginBottom: '0.3rem',
+  },
+});
+
+const sortMembers = (m1: Member, m2: Member) => {
+  return new Date(m2.joinDate).getTime() - new Date(m1.joinDate).getTime();
+};
+
+export const EntityBazaarInfoCard = () => {
+  const { entity } = useEntity();
+  const classes = useStyles();
+  const bazaarApi = useApi(bazaarApiRef);
+  const identity = useApi(identityApiRef);
+  const [anchorEl, setAnchorEl] = useState<HTMLButtonElement>();
+  const [open, setOpen] = useState(false);
+  const [popoverOpen, setPopoverOpen] = useState(false);
+  const [openDelete, setOpenDelete] = useState(false);
+
+  const [isMember, setIsMember] = useState(false);
+  const [members, setMembers] = useState<Member[]>([]);
+  const [bazaarProject, setBazaarProject] = useState<BazaarProject>({
+    entityRef: '',
+    name: '',
+    community: '',
+    announcement: '',
+    status: 'proposed',
+    updatedAt: '',
+  });
+  const [isBazaar, setIsBazaar] = useState(false);
+
+  const getInitMemberStatus = async () => {
+    const response = await bazaarApi.getMembers(entity);
+    const dbMembers = response.data.map((obj: any) => {
+      const member: Member = {
+        userId: obj.user_id,
+        entityRef: obj.entity_ref,
+        joinDate: obj.join_date,
+      };
+
+      return member;
+    });
+
+    dbMembers.sort(sortMembers);
+
+    setMembers(dbMembers);
+    setIsMember(
+      dbMembers
+        .map((member: Member) => member.userId)
+        .indexOf(identity.getUserId()) >= 0,
+    );
+  };
+
+  const getMetadata = async () => {
+    const response = await bazaarApi.getMetadata(entity);
+
+    if (response.status !== 404) {
+      setIsBazaar(true);
+      const data = await response.json().then((resp: any) => resp.data);
+
+      setBazaarProject({
+        entityRef: data[0].entityRef,
+        name: data[0].name,
+        community: data[0].community,
+        announcement: data[0].announcement,
+        status: data[0].status,
+        updatedAt: data[0].updatedAt,
+      });
+    }
+  };
+
+  const { loading } = useAsync(async () => {
+    await getInitMemberStatus();
+    await getMetadata();
+  });
+
+  const onOpen = (event: React.SyntheticEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+    setPopoverOpen(true);
+  };
+
+  const closeEdit = () => {
+    setOpen(false);
+  };
+
+  const closeDelete = () => {
+    setOpenDelete(false);
+  };
+
+  const popoverCloseHandler = () => {
+    setPopoverOpen(false);
+  };
+
+  const handleMembersClick = async () => {
+    setIsMember(!isMember);
+
+    const newMember: Member = {
+      userId: identity.getUserId(),
+      entityRef: stringifyEntityRef(entity),
+      joinDate: new Date().toISOString(),
```

Comment:
the database should probably just automatically set this to the default now() instead, rather than the user supplying it (perhaps wrongly) from the outside